.github/workflows/Dockerfile.multiarch

```diff
@@ -4,9 +4,12 @@ RUN curl https://apt.corretto.aws/corretto.key > corretto.key \
     && apt-key add corretto.key \
     && add-apt-repository 'deb https://apt.corretto.aws stable main' \
     && apt update \
-    && apt install -y cmake clang java-11-amazon-corretto-jdk \
+    && apt install -y cmake clang java-11-amazon-corretto-jdk g++-7 \
+    && find /usr/lib/gcc/aarch64-linux-gnu/ -maxdepth 1 -mindepth 1 ! -name 7 -exec rm -rf {} + \
```

Comment:
What does it to? Maybe a comment could be useful 😅

Reply:
Hmm, I thought it was kinda self explenatory, but sure I can add a comment :+1:

Reply:
Sorry I don't speak bash :D. Also make sure to explain why it is needed.